packages/common/src/utils/stream-utils.ts

```diff
@@ -138,10 +138,12 @@ export class StreamUtils {
   }
 
   /**
-   * Deserializes stream cloned from over the network transfer
+   * Deserializes stream cloned from over the network transfer. Returns null if given null as a param.
    * @param state - Stream cloned
    */
-  static deserializeState(state: any): StreamState {
+  static deserializeState(state: any | null): StreamState | null {
```

Comment:
```suggestion
  static deserializeState(state: any): StreamState | null {
```

as `any` includes `null`